MM-31101. Reworked script name validation page.

rachelbritton opened the pull request:
Restructured and edited the script name validation page.

@mmOrvar, please review the content to identify what might be outdated or incorrect, or might simply not belong in this documentation.

rachelbritton committed:
MM-31101. Reworked script name validation page.

Restructured and edited the script name validation page.

mmOrvar commented on guides/admin-manual/admin_script_name_validation_policy.rst:
```diff
@@ -6,26 +6,35 @@
+**Restrictions**:
 
-* **Scripts Folder**: The changes introduce a new concept of a **scripts folder**. This folder, named ``scripts``, is located in Central's data folder. Existing scripts need to be moved to this folder, and all new scripts should be placed here as well. After moving existing scripts, ensure that field values referring to them are updated accordingly.
+Scripts cannot be have paths outside of the ``scripts`` folder or any of its sub-folders. Validation checks ensure that the script file exists on the server. 
```
Hmm? Cannot be have paths?

mmOrvar commented on guides/admin-manual/admin_script_name_validation_policy.rst:
```diff
@@ -6,26 +6,35 @@
+The script name field is also located in the :guilabel:`Schedule` dialog for report definitions on the **Reports** page.
 
-In version 10.5.5, significant changes have been made to the validation of script name fields for schedules scripts, change event scripts, subnet monitoring scripts, and report results scripts. 
+Since version 10.5.5, significant changes have been made to the validation of script name fields for scheduled scripts, change event scripts, subnet monitoring scripts, and report results scripts. 
```
The change were made in v10.5.5, nothing has been changed since then.

Maybe no need to mention the changes that were done then and only explain the requirements as they are now.

rachelbritton commented on guides/admin-manual/admin_script_name_validation_policy.rst:
```diff
@@ -6,26 +6,35 @@
+Scripts cannot be have paths outside of the ``scripts`` folder or any of its sub-folders. Validation checks ensure that the script file exists on the server. 
 
-* **Restrictions**: The new policy does not allow scripts with paths outside of the scripts folder (or its sub-folders). Validation checks ensure that the script file exists on the server. Additionally, specifying an interpreter as part of the script name value is no longer allowed. For Windows, the interpreter is determined by the script file’s extension, while for Linux, shebang should indicate ...
+You cannot specify an interpreter as part of the script name value. On Windows, the interpreter is determined by the script file's extension. On Linux, a ``shebang`` indicates the interpreter.
```
@mmOrvar, Ármann asked whether this content belongs here?

rachelbritton committed:
Adjusted text.

Adjusted text based on feedback.

rachelbritton committed:
MM-31101. Removed script name validation policy.

Removed script name validation policy page, which is only needed to explain changes in 10.5.5. Unnecessary for versions starting with 11.0.
